.editorconfig

```diff
@@ -0,0 +1,7 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
+end_of_line = crlf
```

Comment:
in that case we can either ignore it (I'll remove the eol from the editorconfig), or we can ensure files are checked out with `lf`, because currently it's forced to autocrlf on windows due to the `.gitattributes` config (which takes precedence over global git config, where I have `autocrlf=false`).

We could tweak the `.gitattributes` as follows:

```
*.txt   text eol=lf
*.js    text eol=lf
*.html  text eol=lf
*.md    text eol=lf
*.json  text eol=lf
*.yml   text eol=lf
*.css   text eol=lf
*.svg   text eol=lf
```